Makefile

```diff
@@ -25,6 +26,7 @@ $(foreach dir, $(filter %Top, $(DIRS)), \
 
 # iocBoot depends on all *App dirs
 iocBoot_DEPEND_DIRS += $(filter %App,$(DIRS))
+postinstall_DEPEND_DIRS += iocBoot $(wildcard *App)
```

Comment:
> Isn't it enough that we depend on iocBoot and it depends on all "App" directories?

Yes, it is enough, since `iocBoot` depends on App. I listed it explicitly because we depend directly on an artifact from the `App` directory.

It is okay to simplify this assuming that `iocBoot` dependencies will never change.

>  If not, wouldn't it be best to use the filter command from above, nonetheless?

Indeed. My mistake.